plat/qemu/qemu_console.c

```diff
@@ -18,22 +17,13 @@ void qemu_console_init(void)
 	(void)console_pl011_register(PLAT_QEMU_BOOT_UART_BASE,
 			       PLAT_QEMU_BOOT_UART_CLK_IN_HZ,
 			       PLAT_QEMU_CONSOLE_BAUDRATE, &console);
+
+	console_set_scope(&console.console, CONSOLE_FLAG_BOOT |
+			  CONSOLE_FLAG_RUNTIME);
 #else
 	console_init(PLAT_QEMU_BOOT_UART_BASE,
 		     PLAT_QEMU_BOOT_UART_CLK_IN_HZ,
 		     PLAT_QEMU_CONSOLE_BAUDRATE);
 #endif /* MULTI_CONSOLE_API */
 }
 
-void qemu_crash_console_init(void)
-{
-#if MULTI_CONSOLE_API
-	(void)console_pl011_register(PLAT_QEMU_CRASH_UART_BASE,
-			       PLAT_QEMU_CRASH_UART_CLK_IN_HZ,
-			       PLAT_QEMU_CONSOLE_BAUDRATE, &crash_console);
-#else
-	console_core_init(PLAT_QEMU_CRASH_UART_BASE,
```

Comment:
Now that you no longer need it, can you please take the console_core_init() prototype you added to include/drivers/console.h back out? I don't think it belongs there, that function was never meant to be called from C (even before MULTI_CONSOLE_API).

Reply:
Fixed